yarn-error.log

```diff
@@ -0,0 +1,112 @@
+Arguments: 
```

Comment:
I think you can get rid of this file 😄 Also, add it to .gitignore so it doesn't happen again.